Makefile

```diff
@@ -20,6 +20,10 @@ qubes-rpc: prep-salt ## Places default deny qubes-rpc policies for sd-svs and sd
 	sudo qubesctl top.enable sd-dom0-qvm-rpc
 	sudo qubesctl --show-output --targets sd-dom0-qvm-rpc state.highstate
 
+sys-usb: prep-salt ## Adds udev rules and scripts to sys-usb
```

Comment:
I think we'll also want to add `sys-usb` to `scripts/list-vm` such that `make all` runs this target. I (otherwise successfully!) provisioned on this but noticed the customizations were missing in `sys-usb` until I ran `make sys-usb` separately

Reply:
I added `sys-usb` to `scripts/provision-all`.